web/client/components/styleeditor/PropertyField.jsx

```diff
@@ -8,8 +8,9 @@
 
 import React from 'react';
 import Message from '../I18N/Message';
+import InfoPopover from '../widgets/widget/InfoPopover';
```

Comment:
this file is missing in original PR is because stable branch had some missing code due to other PRs not being backported?